themes/dark/yaru-dark.json

```diff
@@ -356,9 +356,9 @@
     "list.inactiveSelectionBackground": "#575757",
     "activityBar.background": "#575757",
     "activityBar.dropBackground": "#e9542080",
-    "activityBarBadge.background": "#e95420",
+    "activityBarBadge.background": "#0c6d1a",
     "activityBarBadge.foreground": "#ffffff",
-    "badge.background": "#e95420",
+    "badge.background": "#991121",
```

Comment:
This looks good, but I'm curious about the rationale? I don't think I've seen badges using this color while using Ubuntu.

Reply:
This change is more to the nature of that green in yaru. We use it as a "success" color, i.e. for new things that are not negative. That badges in the bottom panel are generally more of "attention!" things, mostly errors or warnings. So iff they are green it is the wrong message: success, you have an error :smile: 
Good Idea with the contrast, I'll change that for the dark theme!

Reply:
Ah, that definitely makes sense for the problems pane! They also show up on the search results pane, which is why I asked, but I think it's fine.